README.md

```diff
@@ -1,9 +1,346 @@
-OpenMRS Module Appointments Backend
-=================================
+# OpenMRS Module Appointments Backend
+
 This repository acts as the back end for the **Bahmni Appointment Scheduling**.
 
 ## Packaging
-```mvn clean package```
+
+`mvn clean package`
 
 The output is the OMOD file:
-```openmrs-module-appointments/omod/target/appointments-[VERSION].omod```
+`openmrs-module-appointments/omod/target/appointments-[VERSION].omod`
+
+# Setup instructions
+
+This module is incompatiable with the openmrs-appointment-scheduling module, please ensure you remove it either from the admin-page or running this command below from where your modules are located in tomcat directory i.e `/var/lib/OpenMRS/modules` on linux system
+
+```
+sudo rm appointmentscheduling-1.13.0.omod
+```
+
+In order to avoid conflicting tables in the database, please drop the tables below. by running the following sql statements.
```

Comment:
I am not sure if removing the database tables are needed at all. But please add appropriate messaging (as indicated above). Maybe just add 

"You may optionally choose to remove the database tables specific to openmrs-appointment-scheduling".